incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,19 @@
+//////////  S V G   M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
```

Comment:
Yeah. This library is a bit of a 'black-box' in terms of understanding when or why you might want to use  alot of these things. Docs and comments will help a lot :)